src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrStyles = sourceString.split(';');
+
+  const objStyles = {};
+
+  for (const style of arrStyles) {
+    const nameStyle = style.split(':');
+
+    if (nameStyle[0].trim() !== '') {
+      objStyles[nameStyle[0].trim()] = nameStyle[1].trim();
+    }
```

Comment:
You can simplify this logic like this
```suggestion
  const [key, value] = style.split(':').map(item => item.trim());

  if (key) {
    objStyles[key] = value;
  }
```